helm_deploy/values-dev.yaml

```diff
@@ -31,6 +31,3 @@ generic-service:
 
 generic-prometheus-alerts:
   alertSeverity: ppud-replacement-alerts
-  extraDashboardTags:
```

Comment:
Unrelated clean-up?

Reply:
Yes, seemed pointless having a separate PR for it...